plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1710,14 +1711,28 @@ public void testStatsSplitPruningBasedOnSepCreatedCheckpointOnTopOfCheckpointWit
     public void testVacuum()
             throws Exception
     {
+        testVacuumWithCheckPointInterval(false);
+    }
+
+    @Test
+    public void testVacuumWithParquetTransactionLogging()
```

Comment:
Parquet is used for both data files and checkpoints. Please rename to `testVacuumTransactionLoggingOnCheckpoint` or something.